site/content/en/blog/releases/1.23.0-rc.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "1.23.0 - Release Candidate"
+linkTitle: "1.23.0-rc"
+date: "2022-05-03"
+---
+This is the 1.23.0-rc release of Agones.
+
+This release introduces initial support for running game servers on linux machines using the ARM architecture! If this is of interest to you, please test it out and give us feedback so that we can ensure that it works reliably before promoting the support to stable.
+This release brings us support for Kubernetes 1.22, so if you are installing this release, please remember to upgrade your clusters.
+
+This release has lots of important bug fixes includes:
+
+* Fix for health controller race condition marking healthy pods as unhealthy.
+* Check for DeletionTimestamp of fleet and gameserverset before scaling.
+* GameServer stucks in Shutdown state preventing a rolling update to complete.
+*  Fix for GameServer back to Ready always.
+* Queue updated GameServers with DeletionTimestamp.
+
+Check the <a href="https://github.com/googleforgames/agones/tree/release-1.23.0-rc" data-proofer-ignore>README</a> for details on features, installation and usage.
+
+**Implemented enhancements:**
```

Comment:
Please update this section after re-generating the changelog. Also note that it should include both the enhancements and also the breaking changes (put the breaking changes on top).